packages/react-conformance/src/utils/getComponent.tsx

```diff
@@ -0,0 +1,35 @@
+import { ReactWrapper } from 'enzyme';
+import { Ref, RefFindNode } from '@fluentui/react-component-ref';
+// import { FocusZone } from '@fluentui/react-bindings';
+
+export const getDisplayName = (Component: React.ElementType) => {
+  return (
+    (Component as React.ComponentType).displayName ||
+    (Component as React.ComponentType).name ||
+    (typeof Component === 'string' && Component.length > 0 ? Component : 'Unknown')
+  );
+};
+
+// tslint:disable-next-line:no-any
+export const toNextNonTrivialChild = (from: ReactWrapper, wrapperComponent: React.ElementType | undefined): any => {
```

Comment:
Would be nice if you could figure out the actual return type here (or just don't specify it and it might be inferred correctly)